apps/web/src/pages/Home/IAPHomePage/components/Svg/CloseQuote.tsx

```diff
@@ -1,18 +1,29 @@
 import React from "react";
+import { useIntl } from "react-intl";
 
-const CloseQuote = (props: React.HTMLAttributes<HTMLOrSVGElement>) => (
-  <svg
-    xmlns="http://www.w3.org/2000/svg"
-    fill="none"
-    viewBox="0 0 78 62"
-    {...props}
-  >
-    <path
-      fill="#fff"
-      fillOpacity="0.5"
-      d="M62.608 61.6H46.192L56.56 30.784H46.48V.256h31.104v27.648L62.608 61.6zm-46.08 0H.112L10.48 30.784H.4V.256h31.104v27.648L16.528 61.6z"
-    />
-  </svg>
-);
+const CloseQuote = (props: React.HTMLAttributes<HTMLOrSVGElement>) => {
+  const intl = useIntl();
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 78 62"
+      {...props}
+    >
+      <title>
+        {intl.formatMessage({
+          defaultMessage: "closed quote mark",
+          id: "4dWeWs",
+          description: "Indigenous Apprenticeship open quote text alternative",
+        })}
+      </title>
```

Comment:
I would prefer we do not add any text to these and would rather make them presentational. I don't think telling users there is an open and closed quote mark adds anything and would cause more confusion since it wraps something that is already indicated as a quote thorough the HTML element `blockquote`

Reply:
Gray OK'd removing these.  

To make theme presentational I added  `aria-hidden="true"`
https://css-tricks.com/accessible-svgs/#aa-example-2-standalone-icon-decorative

[rollback div tagging and quotes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8033/commits/0b070e907fb0cfe35fe61bc0e7b05dbdde3476c3)